src/lib/server/editors/getSuggestions.ts

```diff
@@ -0,0 +1,64 @@
+import { OPENAI_API_KEY } from '$env/static/private';
```

Comment:
We need this extra file only if we load suggestions in the +page.server.ts which I don't know if it makes sense.

Reply:
see above